lib/Minz/ExtensionManager.php

```diff
@@ -362,7 +366,10 @@ public static function callHook(string $hook_name, ...$args) {
 			return self::callOneToOne($hook_name, $args[0] ?? null);
 		} elseif ($signature === 'PassArguments') {
 			foreach (self::$hook_list[$hook_name]['list'] as $function) {
-				call_user_func($function, ...$args);
+				$result = call_user_func($function, ...$args);
+				if (isset($result)) {
```

Comment:
```suggestion
				if ($result !== null) {
```

That variable will always be set, by it may be null